wiki/Skinning/en.md

```diff
@@ -97,88 +181,6 @@ The version number, seen in the headings, is what will be used in the `Version`
-
-- **1024x768** (4:3, standard game resolution, the game is based on it)
-- **2048x1536** (4:3, standard game resolution in HD scaling)
-- **1366x768** (16:9, standard game widescreen resolution)
-- **2732x1536** (16:9, standard game widescreen resolution in HD scaling)
-- **1920x1080** (16:9, standard HD resolution)
-
-Images will be adjusted by the game itself to fit resolutions derivating from the ones mentioned above.
-Most of them will be rescaled to fit the playfield or repositioned on different aspect ratios.
-
-### osu!supporter
-
-Currently 3 sprites and 2 sounds can only be modified by [osu!supporter](/wiki/osu!supporter) otherwise the default one will be used.
-These are:
-
-- *menu-background.jpg* (this is the exception, this element MUST be in JPG format)
-- *triangle.png* (currently disabled by the game)
-- *welcome\_text.png*
-- *seeya.wav*
-- *welcome.wav*
-
-## HD Sprites
-
-HD sprites are used in place of the normal sprites on higher resolutions, when present. Compared to normal sprites, HD sprites look much sharper and cleaner on these larger resolutions.
-They get scaled by the game itself automatically to fit the resolution used.
-Resolutions supporting HD sprites begin at a **minimum of 800 pixel in height**.
-HD sprites are tagged with **@2x** at the end of their names.
-
-- Example: cursor**@2x**.png
-
-HD sprites have doubled dimension sizes.
-
-- Example: the normal *hitcircle* sprite has the dimension sizes of **128x128 pixel.**
-  - Its HD sprite *hitcircle@2x* has the dimension sizes of **256x256 pixel.**
-
-Every sprite has an HD counterpart, even all frames in an animation can have HD counterparts.
-As a result, the filesize of the folder or archive will increase due to having more images than normal.
-All HD sprites may also be bigger in filesize due to the fact that the canvas size used is four times bigger compared to the normal sprite.
-There are essentially two resolution modes osu! is using. Each of them prefers one set of sprites.
-The first mode is **LowResolution** while the second mode is **HighResolution.**
-
-- *LowResolution mode* uses the normal sprites and ignores the HD sprites **(SD-resolution skin)**
-- *HighResolution mode* prefers HD sprites and uses normal sprites as a fallback if no HD sprite is available **(HD-resolution skin)**
-
-## Skin Sets
-
-If your **beatmap** uses a skin and has any of the elements included in here, all elements of the set **must be included** even if you need to use default skin elements to complete it. If **all** sets are included, then you may alternatively just force default skin.
-
-The **individual sets** can be found on these pages **divided into each gamemode** and the user interface shared by most gamemodes:
-
-- [Interface Sets](./Interface/)
-- [osu!standard Sets](./osu!/)
-- [osu!taiko Sets](./osu!taiko/)
-- [osu!catch Sets](./osu!catch/)
-- [osu!mania Sets](./osu!mania/)
-- [Sound Sets](./Sounds/)
+- does not require the `skin.ini` file
```

Comment:
according to Haskorion's spreadsheet, _osu!_ is at `Version: 2.5`
https://docs.google.com/spreadsheets/d/1bhnV-CQRMy3Z0npQd9XSoTdkYxz0ew5e648S00qkJZ8/edit#gid=1760011283